style.css

```diff
@@ -431,6 +472,76 @@ textarea {
   margin: 4rem auto 0.3rem;
 }
 
+/* style modal */
+
+.modal {
+  border: 2px solid #ebf0ee;
+  margin: 0 7%;
+  border-radius: 10px;
+  display: none;
+  transition: 0.3s ease-in-out;
+  background-color: white;
+  position: fixed;
+  z-index: 10;
+  top: 0;
+}
```

Comment:
- [ ] As suggested by the previous reviewer, Kindly make sure your modal pop-up is centered on the main page of your project and the background of the main page should be `blurred` once the pop-up is displayed. You can achieve centering of the pop-up by giving the CSS property below 👇 

On mobile view give  ```margin: 15% 7%;```

On Desktop view give : ```margin: 4% auto;```

  - You can achieve the blurring by giving the container that hold the card a ```backdrop-filter: blur(10px);```

Reply:
to be honest i have tried implimenting the blur.. if you could assist would really apreciate it: 
![image](https://user-images.githubusercontent.com/92299875/216159649-9619601d-1ff1-48f1-aa22-bdf345229bee.png)